### PR TITLE
warn on ssr if routes doesnt exists

### PR DESCRIPTION
**Pull request by pYr0x**
this PR send a warning to the user if she / he forget to set up routing with can-route

### CODE REVIEW

File: src/template.txt [hunk 1]
```diff
@@ -417,18 +429,18 @@ define(<%= imports %>, function(<%= args %>){
...
 				canReflect.setKeyValue(viewModel, "statusMessage", "Not found");
 			}
 		}
-
-		route.data = viewModel;
-		route.start();
+		if(!isEmptyObject(route.routes)) {
+			route.data = viewModel;
+			route.start();
+		}
```
Comment (matthewp) [inline src/template.txt]: Does anything bad happen if we do this when there are no routes defined?

Comment (matthewp) [reply src/template.txt]: @pYr0x Would like to get this in. Can we remove this if?

Comment (pYr0x) [reply src/template.txt]: i will remove it..

**Comment by matthewp** [comment]
Closes #153

**Pull request merged**